### DISCUSSION TITLE
Variables on Base layer url template

### DISCUSSION THREAD

**Discussion by PChamorro** [grafana/grafana#38441] [Dashboards, Panels & Visualizations]
Hello everyone,

We tried to use variables into URL template input on Base layer (GeoMap visualization on Grafana 8.1.1) but it detects variables like part of the text.

We tried some syntax variables with the same result.

Text$varnameText
Text${var_name}Text

Use variables into the URL template input can be very helpfull to have an unic dynamic dashboad instead of multiple static dashboards.

Thanks.

**Comment by luizluca** [comment]
Dup https://github.com/grafana/grafana/discussions/46287 and https://github.com/grafana/grafana/discussions/57469?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
